.github/workflows/neptune-client.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest] #, macos-latest, windows-latest]
-        python-version: [3.5, 3.6, 3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Betting on Dzwiedziu agreeing to drop 3.5 support - otherwise I'll have to refactor to not use dataclasses.